qiita_db/sql_connection.py

```diff
@@ -150,9 +152,19 @@ def _open_connection(self):
 
         try:
             self._connection = connect(**args)
-        except Exception as e:
+        except Exception:
             # catch any exception and raise as runtime error
-            raise RuntimeError("Cannot connect to database: %s" % str(e))
+            # attempt to point the user to their config file
+            conf_fp = environ['QIITA_CONFIG_FP']
```

Comment:
this isn't ensured to work. possible for the error to indicate that the qiita config cannot be found either if it doesn't exist?